kinogoteborg/src/app/components/booking/screeningDates.jsx

```diff
@@ -1,22 +1,29 @@
 "use client";
 
 import { useEffect, useState } from "react";
+import { fetchScreeningDates } from "@/scripts/fetchScreeningDates";
 
 export default function ScreeningDates({ id, setSelectedDate, selectedDate }) {
   const [screeningDates, setScreeningDates] = useState([]);
 
   useEffect(() => {
-    fetch(`http://localhost:3000/api/booking/${id}/dates`)
-      .then((res) => res.json())
-      .then((data) => {
+    const fetchedDataForDates = async () => {
+      const fetchedData = await fetchScreeningDates(id);
+
+      if (fetchedData.length === 0) {
+        const noScreeningDates = ["No available dates"];
```

Comment:
This seems reduntant?
cant you just use "setSvreeningDates("No  Available dates") on row 16?

Reply:
No, that does not work because of row 57: 

{screeningDates.map((date, index) => (

Reply:
soooo, set....(["no dates"]) ?